js/main.js

```diff
@@ -97,14 +115,42 @@ function viewSwap(viewName) {
   toggleNoEntries();
 }
 
-const $aElement = document.querySelector('a');
+const $entriesButton = document.querySelector('a');
 
-$aElement.addEventListener('click', function (event) {
+$entriesButton.addEventListener('click', function (event) {
   viewSwap('entries');
 });
 
 const $newButton = document.querySelector('.new-button');
 
 $newButton.addEventListener('click', function (event) {
+  const $newH1 = document.querySelector('.new-h1');
+  $newH1.textContent = 'New Entry';
```

Comment:
You already have a `querySelector` for this, you should just reuse that variable rather than redoing this